packages/@expo/cli/e2e/__tests__/utils.ts

```diff
@@ -93,12 +93,16 @@ export async function createFromFixtureAsync(
     fixtureName,
     config,
     pkg,
+    linkExpoPackages,
+    linkExpoPackagesDev,
   }: {
     dirName: string;
     reuseExisting?: boolean;
     fixtureName: string;
     config?: Partial<ExpoConfig>;
     pkg?: Partial<PackageJSONConfig>;
+    linkExpoPackages?: string[];
```

Comment:
```suggestion
    /**
     * Note, this is linked by installing the workspace folder as dependency directly.
     * This may cause other side-effects, like resolving monorepo dependencies instead of the test project.
     */
    linkExpoPackages?: string[];
```